### DISCUSSION TITLE
profile for two controllers at same time

### DISCUSSION THREAD

**Discussion by L30n4rd0-4rg3nt1n0** [AntiMicroX/antimicrox#1047] [Q&A]
Hi, I have two xbox s/x controllers. I want to assign different keyboard keys to each one, for a two-player game. I also want to make a script that runs my game along with AntiMicroX loading the configuration of each controller automatically.

AntiMicroX shows me a tab for each controller detected. The changes I make in each tab are recorded separately. But on the command line it only allows me to load a single file. Since both controllers are the same model, I understand that I can make a single file and define the configuration for each one in different sets.

I have tried the execution line:
antimicrox --tray --profile ./MyProfile.gamecontroller.amgp --startSet 1 1 --startSet 2 2

I want set 1 to be assigned to controller 1 and set 2 to be assigned to controller 2... But it doesn't work, it only detects the first invocation " --startSet 1 1" and ignores the next one.

How can I get different profiles assigned to different joysticks in one command line?

Suggestions are appreciated

**Comment by sightseeer** [comment]
I have the exact same issue and it is so annoying.

**Comment by TheOpponent** [comment]
I also require this for older games with poor joystick support, and two GP2040-CE controllers.
